messages/cache_settings/messages.go

```diff
@@ -3,67 +3,67 @@ package cache_settings
+    CacheSettingsCreateFlagBrowserCacheSettingsMaxTtl = "Informs Browser Cache Settings configuration maximum TTL" 
+    CacheSettingsCreateFlagAdaptiveDeliveryAction     = "Informs the Cache Settings configuration adaptive delivery action." 
 
     // [ update ]
     CacheSettingsUpdateUsage            = "update [flags]"
-    CacheSettingsUpdateShortDescription = "Modifies a Cache Setting"
-    CacheSettingsUpdateLongDescription  = "Modifies a Cache Setting based on given attributes to be used in edge applications"
-    CacheSettingsUpdateOutputSuccess    = "Updated cache setting with ID %d\n"
+    CacheSettingsUpdateShortDescription = "Updates a Cache Settings configuration"
+    CacheSettingsUpdateLongDescription  = "Updates a Cache Settings configuration based on given attributes to be used in edge applications"
+    CacheSettingsUpdateOutputSuccess    = "Updated a Cache Settings configuration with ID %d\n"
 
     // [ describe ]
     CacheSettingsDescribeUsage               = "describe --application-id <application_id> --cache-settings-id <cache-settings-id> [flags]"
-    CacheSettingsDescribeShortDescription    = "Returns information about a specific cache settings"
-    CacheSettingsDescribeLongDescription     = "Returns information about a specific cache settings, based on a given ID, in details"
-    CacheSettingsDescribeFlagApplicationID   = "Unique identifier for an edge application. The '--application-id' flag is mandatory"
-    CacheSettingsDescribeFlagCacheSettingsID = "Unique identifier for an origin. The '--cache-settings-id' flag is mandatory"
+    CacheSettingsDescribeShortDescription    = "Returns information about a specific Cache Settings configuration"
+    CacheSettingsDescribeLongDescription     = "Returns information about a specific Cache Settings configuration, based on a given ID, in details"
+    CacheSettingsDescribeFlagApplicationID   = "Unique identifier for an edge application. The '--application-id' flag is required"
+    CacheSettingsDescribeFlagCacheSettingsID = "Unique identifier for an origin. The '--cache-settings-id' flag is required"
     CacheSettingsDescribeFlagOut             = "Exports the output to the given <file_path/file_name.ext>"
     CacheSettingsDescribeFlagFormat          = "Changes the output format passing the json value to the flag"
     CacheSettingsDescribeHelpFlag            = "Displays more information about the describe subcommand"
 
     // [ delete ]
     CacheSettingsDeleteUsage               = "delete [flags]"
-    CacheSettingsDeleteShortDescription    = "Deletes an Cache Settings"
-    CacheSettingsDeleteLongDescription     = "Deletes an Caches Settings from the Edge Applications library based on its given ID"
-    CacheSettingsDeleteOutputSuccess       = "Caches settings %d was successfully deleted\n"
+    CacheSettingsDeleteShortDescription    = "Deletes a Cache Settings configuration"
```

Comment:
Removing one of the cache settings. suggestion

Reply:
Having the [docs](https://www.azion.com/en/documentation/products/edge-application/edge-caching/#cache-settings) as reference. A Cache Settings configuration can be created, updated or deleted. Since Cache Service is described there as a service and what is created is a config, I think "Deletes a Cache Settings configuration" makes more sense